requests/pulls and committs.py

```diff
@@ -0,0 +1,71 @@
+import requests
```

Comment:
Your pull request: REQUESTS-1022 Added pulls and commits.py
The prefix of your comment must be from ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'HEXNUMBER']

Your commit: REQUESTS-1022 Added pulls and committs.py
The prefix of your comment must be from ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'HEXNUMBER']
Your commit: REQUESTS-1022 Refactored pulls and committs.py
The prefix of your comment must be from ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'HEXNUMBER']
Action of your comment must be from ['Added', 'Deleted']
Your commit: REQUESTS-1022 Refactored pulls and committs.py
The prefix of your comment must be from ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'HEXNUMBER']
Action of your comment must be from ['Added', 'Deleted']
Your commit: REQUESTS-1022 Refactored pulls and committs.py
The prefix of your comment must be from ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'HEXNUMBER']
Action of your comment must be from ['Added', 'Deleted']